src/stratis_cli/_error_codes.py

```diff
@@ -120,12 +120,76 @@ def from_str(code_str):
         )
 
 
+class PoolDeviceSizeChangeCode(IntEnum):
```

Comment:
This is the only class in `_error_codes` that doesn't follow the `Pool.*ErrorCode` naming pattern (and also the only class to define an I(nfo) as well as W/E). PoolDeviceSizeChangeErrorCode is a bit of a mouthful at 29 chars. It stuck out a bit and I wondered if it might be confusing to come across the class name in isolation but I think I'm probably being nit-picky here :)